scripts/verify-local.sh

```
@@ -0,0 +1,198 @@
+
+    b="$(call_tool_body users_search '{"limit":5}')"
+    printf '%s' "${b}" | grep -q 'UserID,UserName' && ok "#15 users_search returns CSV" || bad "#15 users_search"
+
+    if is_real "${XOXP}"; then
+      b="$(call_tool_body conversations_search_messages '{"query":"the","count":3}')"
+      printf '%s' "${b}" | grep -q 'MsgID,UserID' && ok "#16 search succeeds with xoxp" || bad "#16 search: $(printf '%s' "${b}" | head -c 200)"
+    fi
+
+    if [ -n "${ALLOWED_CHANNEL:-}" ] && [ -n "${DENIED_CHANNEL:-}" ]; then
+      b="$(call_tool_body conversations_add_message "{\"channel_id\":\"${ALLOWED_CHANNEL}\",\"text\":\"slackrabbit verify: allowed channel post\"}")"
+      printf '%s' "${b}" | grep -q 'Successfully posted' && ok "#17 allowed channel: posted" || bad "#17 allowed channel: $(printf '%s' "${b}" | head -c 200)"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)